src/main/java/com/gromoks/filestatistic/entity/FileInfo.java

```diff
@@ -0,0 +1,64 @@
+package com.gromoks.filestatistic.entity;
+
+import java.util.List;
+
+public class FileInfo {
+    private String fileName;
+    private List<String> longestWords;
+    private List<String> shortestWords;
+    private int length;
+    private double averageWordLength;
```

Comment:
mr T. suggested to store LineInfo as field in FileInfo entity for convenieniance